utilities/compute_gc.py

```diff
@@ -113,25 +112,38 @@ def calculate_coverage(bin_dict: dict, window: int) -> float:
     return all_mean / float(running_total)
```

Comment:
We still need to handle Divide_by_Zero error in some cases here @joshfactorial

Reply:
Changed the end of calculate_coverage to:

"""
    if running_total:
        return all_mean / float(running_total)
    else:
        return 0.0
"""